app/xl_NewFileByTemplate.py

```diff
@@ -73,7 +73,7 @@ def SetUp(xlApp):
 			ws = xlApp.Sheets(tempSheetName)
 			ws.Activate
 	        
-			xlApp.ActiveWorkbook.SaveAs(newFilePath + "\\" + str(newFileName[i]) + ".xlsx")
+			xlApp.ActiveWorkbook.SaveAs(newFilePath + "\\" + str(newFileName) + ".xlsx")
 			xlApp.ActiveWorkbook.Close(False)
 			xlApp.screenUpdating = True
 			Marshal.ReleaseComObject(ws)
```

Comment:
That's only true if you are feeding it a single file name and expecting a single template to be processed. This workflow was created to handle lists.